services/blockchain-indexer/shared/indexer/indexStatus.js

```diff
@@ -115,7 +115,6 @@ const init = async () => {
 
 	// Register event listeners
 	Signals.get('newBlock').add(checkIndexReadiness);
-	Signals.get('chainNewBlock').add(updateFinalizedHeight);
```

Comment:
Don't we need to update finalised height after every block?
This below function updates finalised height now. Does it get called after every block? 
https://github.com/LiskHQ/lisk-service/blob/development/services/blockchain-indexer/shared/messageProcessor.js#L156

Reply:
We are already updating finalised height here https://github.com/LiskHQ/lisk-service/blob/1900-lisk-service-performance-issues/services/blockchain-indexer/shared/constants.js#L195-L206